src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -168,19 +234,33 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * As we can see from the configuration, the TranslationCheck was configured
- * to check an existence of 'es', 'fr' and 'de' translations. Let's assume that
- * we have the resource bundle:
+ * Example:
  * </p>
  * <pre>
- * messages_home.properties
- * messages_home_es_US.properties
- * messages_home_fr_CA_UNIX.properties
+ * #messages_es.properties
+ * message_error_es=Ha ocurrido un error
+ * message_warning_es=Advertencia
+ * message_info_es=Información
+ *
+ * #messages_fr.properties
+ * message_error_fr=Une erreur s'est produite
+ * message_warning_fr=Attention
+ * message_info_fr=Information
+ *
+ * #messages_de.properties
+ * message_error_de=Es ist ein Fehler aufgetreten
+ * message_warning_de=Warnung
+ * message_info_de=Information
+ * </pre>
+ * <pre>
+ * # Ok
+ * messages_es.properties
+ * messages_fr.properties
+ * messages_de.properties
+ *
+ * # Violation: missing translation for es, fr and de
+ * messages.properties
```

Comment:
move this Violation situation in separate `pre` tag to show that it will separate execution on same config.
in java code you can mix ok and violation. On different files it is not possible, so it should be visually different codeblocks (`pre` tags).

Reply:
I think we need to make big exception for this Check for general rules on examples and make examples in format of
![image](https://user-images.githubusercontent.com/812984/226908290-9d8c5d08-d020-4b09-9524-ea149d775fca.png)

Reply:
Consider the following properties file in the same directory:
```
some text
```
Validation results:
```
some violation text in brief form
```

Reply:
> I think we need to make big exception for this Check for general rules on examples and make examples in format of ![image](https://user-images.githubusercontent.com/812984/226908290-9d8c5d08-d020-4b09-9524-ea149d775fca.png)

@romani that would be great. but we also have to show the Violation for the files which does not have the extention ".properties, .translations" and the same for all other examples. Please correct me if am wrong.

Reply:
If CLI place violation on them - yes.

Reply:
This is not clear.
Please read example in description , see my screenshot, one more time

Reply:
As per the example you gave.
its looks like this 
```
#messages.properties
hello=Hello
cancel=Cancel

#messages.translations
hello=Hallo
ok=OK
```
```
messages.properties: Key 'ok' missing.
messages.translations: Key 'cancel' missing.
```
is this will be okay ?

Reply:
Looks good, let see how it looks like in html